README.md

```diff
@@ -65,6 +65,9 @@ aa3ed92159be6712badd4d61c089089ba27594456e108c8642e9535e77786562
 - [x] WR841N
 # ffs-30b5c2cb6e56 
 f12fd2fbdc8a97981074d862e0dc928c1968801ce0db9ccc596822ad252133bb
+- [x] WR841ND
+# ffs-abendlaendischer-lebensbaum
+
```

Comment:
(•◡•)/ Das ist der public key. In dem Fall wars: `1e9424aa089842ff2bf465ae10d969b5dc54c11eeee15d5489d5964b5f246876`